docs/faq/README.md

```diff
@@ -1 +1,46 @@
-TBD
+# FAQ
+
+### What operating systems does Lens support?
+
+Lens supports MacOS, Windows and Linux operating systems. For Linux there are Snap and AppImage versions. For MacOS there are DMG and Homebrew options.
+
+### Lens application is not opening, what might be wrong?
+
+When Lens is started, it will start HTTP proxy server on the background and requires that operationg system allows to start listening to some free port. You can see the port allocated for Lens from application logs. Lens expects also that `localhost` DNS points to `127.0.0.1` address.
```

Comment:
```suggestion
When Lens is started, it will start HTTP proxy server on the background and requires that operating system allows to start listening to some free port. You can see the port allocated for Lens from application logs. Lens expects also that `localhost` DNS points to `127.0.0.1` address.
```

Reply:
Fixed